svg.js.d.ts

```diff
@@ -1169,8 +1169,7 @@ declare module "@svgdotjs/svg.js" {
 
         addClass(name: string): this;
         after(element: Element): Element;
-        animate(duration?: number, ease?: string, delay?: number): Runner;
-        animate(info: { ease?: string; duration?: number; delay?: number }): Runner;
+        animate(duration?: number, delay?: number, when?: string): Runner;

```

Comment:
Albeit this is correct, it also allowed for `animate({object})` like:

```
animate({
  duration: ...,
  when: ...,
  delay: ...,
  swing: ...,
  times: ...,
  wait: ...,
})
```

`swing, times and when` are known from the `loop` method

Reply:
Thank you for the review! :+1: I was having trouble finding this signature in the code, but I took another look and I was able to find it in sanitise :-)

I changed both the function signatures to use the `TimeLike` type that was already defined.